content/reference/template-functions/static-context.md

```diff
@@ -159,6 +158,13 @@ Or for a total of 64 `a`s and `b`s:
 '{{repl RandomString 64 "[ab]" }}'
 ```
 
+Each time that this function is called, the behavior changes based on the [hidden](/reference/v1beta1/config/#hidden) and [readonly](/reference/v1beta1/config/#readonly) properties.
+
+- To generate `RandomString` value that is **persistent** between Config changes, use it in conjuction with `hidden` property set to `true`. The `value` is not shown in HTML, hence it cannot be modified.
```

Comment:
```suggestion
- To generate a `RandomString` value that is **persistent** between Config changes, use it in conjunction with `hidden` property set to `true`. The `value` is not shown in HTML, hence it cannot be modified.
```

Reply:
@aj-jester I noticed this was resolved but the change was not applied -- can we fix the typos here?